crates/cheatnet/tests/README.md

```diff
@@ -0,0 +1,2 @@
+Before testing build with:
+`cd contracts && scarb build`
```

Comment:
This should be included somewhere in the development as well. Perhaps we could include this procedure in `scripts/prepare_for_tests.sh` ?

Reply:
Found a better solution